src/Mvc/src/Microsoft.AspNetCore.Mvc.ViewFeatures/Infrastructure/DefaultTempDataSerializer.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Infrastructure
+{
+    internal class DefaultTempDataSerializer : TempDataSerializer
+    {
+        public override IDictionary<string, object> Deserialize(byte[] unprotectedData)
+        {
+            throw new NotSupportedException();
+        }
+
+        public override byte[] Serialize(IDictionary<string, object> values)
+        {
+            throw new NotSupportedException();
+        }
+    }
```

Comment:
???????

Reply:
Need System.Json.JsonSerializer. This will get fixed eventually. But for now, this avoids having to add a reference to NewtonsoftJson package everywhere just because you have a controller \ view